website/pages/component/button/example.tsx

```diff
@@ -9,7 +9,7 @@ const knobsConfig: KnobsConfigInterface<{
   size: Size;
   glyph: string;
   children: React.ReactNode;
-  href: string;
+  href: boolean;
```

Comment:
naming: `href` being a `boolean` is confusing. how about `hasHrefProp`

Reply:
Makes sense, will do.

Reply:
Actually helped me catch another bug, thank you!